pkgs/applications/science/math/sage/sage-with-env.nix

```diff
@@ -13,7 +13,7 @@
 , pari
 , gmp
 , gfan
-, python2
+, python
```

Comment:
```suggestion
, python3
```

Reply:
"python" here is not pkgs.python, because default.nix does
```
  python = pkgs.python3.override {
    packageOverrides = self: super: {
      sagelib = self.callPackage ./sagelib.nix {
...
  }}
  sage-with-env = callPackage ./sage-with-env.nix {
    inherit python pythonEnv;
```
I felt it would be more consistent to use this overriden Python 3 package everywhere instead of using pkgs.python3. Do you think it would be less confusing to rename default.nix's `python` variable to `python3` and pass that along instead?

Reply:
> Do you think it would be less confusing to rename default.nix's python variable to python3 and pass that along instead?

Maybe, as searching `/python[^3]/` is the most easy-to-come-up-with way when one want to find targets for the Python 3 migration.